pic32/libraries/SPI/SPI.h

```diff
@@ -180,10 +183,10 @@ class SPIClass {
 
         inTransactionFlag = 1;
 #endif
-        pspi->sxCon.clr = (1 << _SPICON_ON);
+        // This causes a nasty spike
+        //pspi->sxCon.clr = (1 << _SPICON_ON);
```

Comment:
Nah, don't worry about it. I understand the logic of the changes now, and I'm going to merge the PR, then I want to take a little time this weekend to make a few minor tweaks myself and test them out to make this section of code even clearer. Thanks again - taking the time to submit PRs like this is _so_ appreciated!